app/services/git/repository.rb

```diff
@@ -45,6 +45,10 @@ def valid?
     provider.valid?
   end
 
+  def init_from_template(name)
+    provider.init_from_template(name)
```

Comment:
il faut le miroir sur gitlab si tu l'appelles comme ça, non ?

Reply:
A l'occasion oui, je vais mettre un not implemented